.github/workflows/yarn_test.yaml

```diff
@@ -31,6 +31,10 @@ jobs:
         run: |
           cd packages/noir-ethereum-api
           yarn typecheck
+      - name: Run linter
+        run: |
+          cd packages/noir-ethereum-api
```

Comment:
Maybe use
```yaml
    defaults:
      run:
        working-directory: ./packages/noir-ethereum-api
```
in job definiotion to removew duplications `cd packages/noir-ethereum-api`